### PR TITLE
Heroku deployment

### PR DESCRIPTION
**Pull request by mubstimor**
**What does this PR do?**

Deploys frontend to heroku

**Description of Task to be completed?**

- Change server build commands.

**How should this be manually tested?**

- Visit live app [here](https://ah-frontend-valkyrie.herokuapp.com)

### CODE REVIEW

**Comment by fatukunda** [comment]
@mubstimor I cannot access the app when I click on the link. Is it working as of right now? Could you also include the PT story for this chore?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 node_modules/
-dist/
+dist/
```
Comment (fatukunda) [inline .gitignore]: Please provide a line at the end.

File: package.json [hunk 2]
```diff
@@ -5,17 +5,20 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --open --mode development",
-    "build": "webpack --mode production"
+    "dev": "webpack-dev-server --open --mode development",
+    "start": "serve -s build",
```
Comment (fatukunda) [inline package.json]: You seem to be changing our start command. This means we won't be able to run `yarn start` to run the app locally. Will the serve command do a similar job?

File: .gitignore (hunk 1 above)
Comment (mubstimor) [reply .gitignore]: resolved.

File: package.json (hunk 2 above)
Comment (mubstimor) [reply package.json]: yes, yarn start should be able to work locally, serve is For Heroku production deployment.

**Pull request merged**